### PR TITLE
small improvements

### PR DESCRIPTION
**Pull request by Grzesiek0301**

### CODE REVIEW

File: src/main/java/com/DoctorList.java [hunk 1]
```diff
@@ -8,19 +8,21 @@
...
-		while (true) {
-			System.out.println("What do you want to do?");
-			System.out.println("1.Add the doctor.");
-			System.out.println("2.Remove the doctor.");
-			System.out.println("3.Show list of doctors.");
-			System.out.println("4.Exit ");
-			System.out.println("");
-			System.out.println("Choose number from 1 to 4: ");
+		while (loopCondition == true) {
+			StringBuilder printMainMessage = new StringBuilder();
```
Comment (uki47) [inline src/main/java/com/DoctorList.java]: It's not a method, therefore you should use nouns to name variables.
Please rename it to mainMessage.

File: src/main/java/com/Doctor.java [hunk 2]
```diff
@@ -1,26 +1,32 @@
 package com;
 
 public class Doctor {
-	private final int id;
+	private final int personalDoctorId;
 	private final String name;
 	private final String surname;
+	private final String medicalSpecializiation;
```
[the remark this answers is not in the document]
Comment (uki47) [reply src/main/java/com/Doctor.java]: It would be less error prone.

File: src/main/java/com/DoctorList.java [hunk 3]
```diff
@@ -8,19 +8,21 @@
...
+					.append("1.Add the doctor.\n")
+					.append("2.Remove the doctor.\n")
+					.append("3.Show list of doctors.\n")
+					.append("4.Exit \n");
+			System.out.println(printMainMessage);
 
-			int read;
 			Scanner write1 = new Scanner(System.in);
-			read = write1.nextInt();
+			int read = write1.nextInt();
```
Comment (uki47) [inline src/main/java/com/DoctorList.java]: Raname write1 to scanner or input. Current name is less trivial.

File: src/main/java/com/Doctor.java [hunk 4]
```diff
@@ -1,26 +1,32 @@
 package com;
 
 public class Doctor {
-	private final int id;
+	private final int personalDoctorId;
```
[the remark this answers is not in the document]
Comment (Grzesiek0301) [reply src/main/java/com/Doctor.java]: You're right !

**Pull request closed without merging**